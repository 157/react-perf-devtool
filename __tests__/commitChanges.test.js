const MEASURES = require('../samples/measures')

const getCommitChangesTime = require('../src/utils/commitChanges')
const getTotalTime = require('../src/utils/totalTime')

describe('Committing changes time duration', () => {
  test('Committing changes with and without warning for cascading updates in earlier commits', () => {
    expect(
      getCommitChangesTime(MEASURES)
    ).toMatchSnapshot()
  })

  console.log(getTotalTime(getCommitChangesTime(MEASURES)))
  test('Returns total time taken in committing changes', () => {
    expect(getTotalTime(getCommitChangesTime(MEASURES))).toMatchSnapshot()
  })
})
