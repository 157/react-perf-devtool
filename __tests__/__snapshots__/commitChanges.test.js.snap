// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Committing changes time duration Committing changes with and without warning for cascading updates in earlier commits 1`] = `
Object {
  "Committing Changes": Object {
    "timeSpent": Array [
      4.32,
      1.88,
    ],
  },
}
`;

exports[`Committing changes time duration Returns total time taken in committing changes 1`] = `6.2`;
